applications/StructuralMechanicsApplication/python_scripts/structural_mechanics_solver.py

```diff
@@ -486,7 +487,7 @@ def _create_newton_raphson_strategy(self):
         linear_solver = self.get_linear_solver()
         mechanical_convergence_criterion = self.get_convergence_criterion()
         builder_and_solver = self.get_builder_and_solver()
-        return KratosMultiphysics.ResidualBasedNewtonRaphsonStrategy(computing_model_part,
+        strategy = KratosMultiphysics.ResidualBasedNewtonRaphsonStrategy(computing_model_part,
```

Comment:
Please use Parameters as input instead of adding more and more arguments to the constructor

Reply:
I agree about using parameters, however for now it is not used.
As soon as the factories are in i will also push for removing the other constructors